### PR TITLE
👷 bump json5 resolutions

### PR DESCRIPTION
**Pull request by bcaudan**
## Motivation

Fix security issue raised by dependabot

## Changes

Pin json5 resolution to 1.0.2 and 2.2.2

## Testing

- [ ] Local
- [ ] Staging
- [ ] Unit
- [ ] End to end

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: test/app/yarn.lock [hunk 1]
```diff
@@ -474,11 +474,9 @@ json-schema-traverse@^0.4.1:
   integrity sha512-xbbCH5dCYU5T8LcEhhuh7HJ88HXuW3qsI3Y0zOZFKfZEHcpWiHU/Jxzk629Brsab/mMiHQti9wMP+845RPe3Vg==
 
 json5@^1.0.1:
-  version "1.0.2"
-  resolved "https://registry.yarnpkg.com/json5/-/json5-1.0.2.tgz#63d98d60f21b313b77c4d6da18bfa69d80e1d593"
-  integrity sha512-g1MWMLBiz8FKi1e4w0UyVL3w+iJceWAFBAaBnnGKOpNa5f8TLktkbre1+s6oICydWAm+HRUGTmI+//xv2hvXYA==
-  dependencies:
-    minimist "^1.2.0"
+  version "2.2.2"
```
Comment (BenoitZugmeyer) [inline test/app/yarn.lock:477]: **🔨 warning:** ‏The pinned version does not match the requirement here. When installing the test app dependency, yarn change it back to `1.0.2`.

Because the vulnerability has been fixed in `1.0.2`, I think it's fine to revert the changes from `test/app/yarn.lock`.

Comment (bcaudan) [reply test/app/yarn.lock]: Indeed 🤦‍♂️
